tests/python/test_bls.py

```diff
@@ -60,88 +60,89 @@ def copy():
 
```

Comment:
Again GitHub gets confused about the change, probably because I moved `_test_bls_stencil` to a different file.

The changed functions are `test_scatter_1d_trivial`, `test_scatter_1d`, and `test_multiple_inputs`. Feel free to ignore others.